Sources/MountebankSwift/Models/Imposter/Imposter.RecordedRequest.swift

```diff
@@ -1,13 +1,13 @@
 import Foundation
 
 extension Imposter {
-    public struct RecordedRequest: Equatable, Codable, CustomDebugStringConvertible {
+    public struct RecordedRequest: Equatable, Codable {
         public let method: HTTPMethod?
         public let path: String?
         public let query: [String: String]?
-        public let headers: [String: String]?
+        public let headers: [String: JSON]?
```

Comment:
This needs to be JSON, during testing with some app this crashed when a header contained an Int instead of a string..

Reply:
Should a header not aways be handled as a String? And the parsing to Int should be done later on?

Reply:
I think you're right. https://stackoverflow.com/a/34153120/672989

But the decode was failing because some recorded request in mountebank contained an `Int` instead. Weird right. 

ok, lets revert this, I'll fix it later